www/include/monitoring/external_cmd/popup/popup.php

```diff
@@ -36,14 +36,13 @@
 require_once realpath(dirname(__FILE__) . "/../../../../../config/centreon.config.php");
 
 require_once _CENTREON_PATH_ . 'vendor/autoload.php';
-
+require_once $centreon_path . 'bootstrap.php';
```

Comment:
This code is functional, but to improve the code, it's better to remove the line 
`require_once realpath(dirname(__FILE__) . "/../../../../../config/centreon.config.php");`
And change the path of require "bootstrap.php" whith the relative path.
Because bootstrap.php allready load the configurations files.